infra/data-pipeline.yml

```diff
@@ -27,7 +27,7 @@ tasks:
     routes:
       - notify.email.release-mgmt-analysis@mozilla.com.on-failed"
       - notify.irc-channel.#bugbug.on-failed
-      - index.project.relman.bugbug.data_commits.latest
+      - index.project.relman.bugbug.data_commits.${version}
```

Comment:
We should index both on latest and on ${version}, so for bugbug we use versioned indexes, and other users might want to always use latest.